src/models/panelSet.model.ts

```diff
@@ -0,0 +1,32 @@
+import {
+    CreationOptional, DataTypes, InferAttributes, InferCreationAttributes, Model, ModelStatic, Sequelize
+} from 'sequelize';
+
+interface IPanelSet extends Model<InferAttributes<IPanelSet>, InferCreationAttributes<IPanelSet>> {
+    id: CreationOptional<number>,
+    author_id: string
+}
+
+let PanelSet: ModelStatic<IPanelSet>;
```

Comment:
Now that we're feeding sequelize objects into our queries we don't need to export the model anymore from `.model.ts` files. (Check Arthur's PR for what he removes #5 )